### PR TITLE
Moved stop tag handling to separate flow where it can be handled appr…

### PR DESCRIPTION
**Pull request by karl-exini**
…opriately

### CODE REVIEW

**Comment by karl-exini** [comment]
closes #6

File: src/main/scala/com/exini/dicom/streams/DicomFlows.scala [hunk 1]
```diff
@@ -48,6 +48,22 @@ object DicomFlows {
     a
   }
 
+  /**
+    * Flow for stopping processing early once a tag has been reached. Attributes in sequences are ignored.
+    *
+    * @param tag tag number referring to the root dataset at which the processing stop (exclusive)
+    * @return the associated flow
+    */
+  def stopTagFlow(tag: Int): PartFlow =
```
Comment (fredrik-exini) [inline src/main/scala/com/exini/dicom/streams/DicomFlows.scala:57]: Have using a TagPath instead of Int been considered?

**Review by fredrik-exini — APPROVED** [review]

File: src/main/scala/com/exini/dicom/streams/DicomFlows.scala (hunk 1 above)
Comment (karl-exini) [reply src/main/scala/com/exini/dicom/streams/DicomFlows.scala:57]: Great point. Yes I thought about it but figured that this flow is used when you care about performance, and when profiling it seems that creating a tag path for every dicom part puts some stress on the GC. Do you see any obvious use cases where you would like to stop in the middle of a sequence?

Comment (fredrik-exini) [reply src/main/scala/com/exini/dicom/streams/DicomFlows.scala:57]: No, not really.

**Pull request merged**